sources/models/shared-defaults/kubernetes-services.toml

```diff
@@ -7,6 +7,7 @@ configuration-files = [
   "kubelet-exec-start-conf",
   "kubernetes-ca-crt",
   "proxy-env",
+  "credential-provider-config-yaml",
```

Comment:
This needs a `ReplaceListsMigration` to fix up the existing list on upgrade, and restore it on downgrade.